api-reference/10 UI Components/Widget/1 Configuration/activeStateEnabled.md

```diff
@@ -5,7 +5,9 @@ default: false
 ---
 ---
 ##### shortDescription
-Specifies whether or not the UI component changes its state when interacting with a user.
+Specifies whether the UI component changes its state as a result of user interaction.
 
 ---
-This property is used when the UI component is displayed on a platform whose guidelines include the active state change for UI components.
+The UI component switches to active state when users press down the primary mouse button. When this property is set to **true**, the CSS rules for the active state apply. You can change these rules to customize the component.
+
+Use this property when you display the component on a platform whose guidelines include the active state change for UI components. See the example of such platform: [MUI](https://github.com/mui/material-ui).
```

Comment:
```suggestion
Use this property when you display the component on a platform whose guidelines include the active state change for UI components. See the following topic for an example of this type of platform: [MUI](https://github.com/mui/material-ui).
```

Reply:
I think that "topic" is not the right word, because the link goes to the official github repository of the platform, not the article. Can I write "See the following link" or something like that?

Reply:
Thanks for the clarification. How about the following?

`Use this property when you display the component on a platform whose guidelines include the active state change for UI components. See the following GitHub repository for an example of this type of platform: [MUI](https://github.com/mui/material-ui).`